### PR TITLE
Removes skull balaclava and skull facepaint from loadout, places them hidden on the Almayer.

### PR DESCRIPTION
**Pull request by NewyearnewmeUwu**
# About the pull request

This removes the skull balaclavas and the facepaints from the loadout menu and instead places them in 2 places hidden around the Almayer. The reason I  have done this is that they are almost exclusively used by people who who are referencing a character- usually Ghost from MW2 (either version) or the characters from COD Ghosts. See below for more details.

# Explain why it's good for the game

This is an OOC meme item that doesn't fit the tone of CM, particularly because we _already_ have an item with a skull on it in case you want to use it: Armor! They wrote things on armor in the movie, including a skull!
image

Instead, we have these types of people, running a skull 'clava in every round even as command or medical characters. This is a modern 'operator' look, not a Space 'Nam-esque look and not an Aliens look. If you want something that'd remind you of Space 'nam, just look at the classic 'born to kill' helmet. Now, look at these CALL OF DUTY CHARACTERS THAT THIS ITEM REFERENCES!

image
image
image

This is goofy as hell and kind of an outlier among the customization options since it is _very clearly_ a reference to COD. Look at its description: 

"The face of your nightmares. Heed the call of duty as the ghost in the night with this metal 'clava. Additionally protects against the cold. Now in black!"

You'd get laughed off a real marine base for wearing this, let alone wearing one on an op. We don't need more people running this every round, and this gives them something to fight over between eachother- if _you_ want to larp as Simon 'Ghost' Riley from hit video game COD MW2 (either version) then you'll have to hunt it down.

# Testing Photographs and Procedure

<details>
<summary>Screenshots & Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

# Changelog

:cl:
del: Removed skull balaclavas and skull facepaint from the loadout options
maptweak: adds a single skull facepaint and balaclava, each hidden in their own locations on the Almayer.
/:cl:

### CODE REVIEW

**Comment by NewyearnewmeUwu** [comment]
> PR closed and they just _open a new one_

I wanted to more succintly state my point.

**Comment by stanalbatross** [comment]
ngl your justification is very good. I agree with it and support this PR

**Review by stanalbatross — COMMENTED** [review]
its not hidden when you can look up where it is and beeline to it every round

make it random at least

**Comment by NewyearnewmeUwu** [comment]
Random spawns are in. Don't have an icon for them as of yet so I reused the colony goggle loot spawner icon.

**Review by harryob — APPROVED** [review]
code b fine

File: code/game/objects/effects/spawners/random.dm [hunk 1]
```diff
@@ -136,6 +136,33 @@
...
+	icon_state = "loot_goggles"
+
+/obj/effect/spawner/random/balaclavas/item_to_spawn()
+	return pick(prob(100);/obj/item/clothing/mask/balaclava,\
+				prob(50);/obj/item/clothing/mask/balaclava/tactical,\
+				prob(25);/obj/item/clothing/mask/rebreather/scarf/green,\
+				prob(25);/obj/item/clothing/mask/rebreather/scarf/gray,\
+				prob(25);/obj/item/clothing/mask/rebreather/scarf/tan,\
+				prob(10);/obj/item/clothing/mask/rebreather/skull,\
+				prob(10);/obj/item/clothing/mask/rebreather/skull/black)
```
Comment (stanalbatross) [inline code/game/objects/effects/spawners/random.dm:151]: will these probs actually work?

**Review by stanalbatross — CHANGES_REQUESTED** [review]
also give them chances to spawn nothing (high)

File: code/game/objects/effects/spawners/random.dm (hunk 1 above)
Comment (harryob) [reply code/game/objects/effects/spawners/random.dm:151]: yeah it's weird dm syntax but the weighting numbers are kinda arbitrary anyway

**Comment by NewyearnewmeUwu** [comment]
> also give them chances to spawn nothing (high)

Done. 50% chance to not spawn anything.

**Review by stanalbatross — APPROVED** [review]
code fine

File: maps/map_files/USS_Almayer/USS_Almayer.dmm [hunk 2]
```diff
@@ -71492,6 +71495,7 @@
 /area/almayer/shipboard/navigation)
 "sti" = (
 /obj/structure/closet/crate/trashcart,
+/obj/item/clothing/mask/rebreather/skull/black,
```
Comment (stanalbatross) [inline maps/map_files/USS_Almayer/USS_Almayer.dmm:71498]: remove the confirmed one

**Review by stanalbatross — CHANGES_REQUESTED** [review]

**Review by stanalbatross — APPROVED** [review]

**Comment by 13lacklight** [comment]
Why on earth was this put through? We are USCM, Colonial MARINES, are you telling me there isnt gonna be 1 marine anywhere who decides to wear a Skull balaclava or mask or anything? Like these are soldiers you see wearing and using like a hundred different items of customisation. This is a joke tbh. Are we gonna remove the ace of spades next cause its an OOC vietnam war reference? Shall we remove the Space Jam tickets? Or our Uno reverse cards? 

The whole idea behind this PR is just stupid tbh. Skull facepaint is 110% something id bet you could find Vietnam era troops and shit wearing. a skull balaclava might be slightly pushing it but like really? its a scrap of cloth.
Also i like how your examples of how its only related to modern operators is just 3 posters of call of duty. like what the fuck.
And its not even like its a blindingly obvious or unsettling or stand out cosmetic item. This is just some Github princess who hates people having fun in his game.

Also if youre gonna remove shit, how about you make a replacement first, instead of getting your knickers in a twist and doing nothing but removing content you dont like.

anyway, rant over, but this is such a stupid change, i dont even care that much, i have 1 charachter that used to use it cause it looked nice while wearing the USCM hat, and it was just my chill rifleman charachter, but to remove it was just a weird, strange and stupid change. It annoys me that it was changed for no good reason, not that it was changed. 
Like of all the things you could be doing, youre going through and selectively removing cosmetics that you dont like the look of cause you feel like it. It just comes off as childish, and ya wonder why people complain about yall being completely detached from the player base 9 times in 10.

Itd be nice if you reversed it, but i doubt you will, if youre not going to, it would be great to see some other full face coverings that arent just a single bland colours. Yes we have vietnam aesthetic, but we arent in space vietnam, lord knows we have enough shit that youd already get laughed out of a real marine corps for wearing.

**Pull request merged**